extension/src/test/suite/plots/index.test.ts

```diff
@@ -259,6 +260,158 @@ suite('Plots Test Suite', () => {
       })
     })
 
+    describe('Bounding Box Plots', () => {
+      it('should handle image by step plots with bounding boxes', async () => {
+        const multiImgDirPath = join('plots', 'image')
+        const multiImgPath = join('plots', 'image', '5.jpg')
+        const mockBoxes = {
+          car: [{ box: { bottom: 0, left: 0, right: 0, top: 0 }, score: 0.5 }]
+        }
+        const imgDataWithBoundingBoxes = plotsDiffFixture.data[
```

Comment:
Another option is adding a new image by step plot with bounding boxes plot to our base comparison fixture.